atom/common/node_bindings.h

```diff
@@ -25,7 +25,7 @@ namespace atom {
 
 class NodeBindings {
  public:
-  enum BrowserEnvironment {
+  enum class BrowserEnvironment {
```

Comment:
Maybe just `BrowserEnvironment`=>`Environment` is better

Reply:
done

Reply:
actually, it might get confused with `base::Environment`, let's keep it `BrowserEnvironment`

Reply:
I don't think it should, when they are being qualified by namespace. `BrowserEnvironment::Renderer`, `BrowserEnvironment::Worker` doesn't read correct comparatively. I would have suggested `NodeEnvironment` but that is more likely to be confused with `node::Environment`.